apps/nuxt3-ssr/components/filter/Sidebar.vue

```diff
@@ -41,7 +41,7 @@ function handleFilerUpdate(filter: IFilter) {
       <FilterContainer
         v-for="filter in filters"
         :title="filter.config.label"
-        :conditions="filter.conditions"
+        :conditions="(filter as IOntologyFilter | IRefArrayFilter).conditions"
```

Comment:
using 'as' is a bit of a smell, either the typing of filter is not set correctly or the typing of the conditions prop.